src/CloudSecrets.UnitTests/SecretsToAnalysisIssueConverterTests.cs

```diff
@@ -91,14 +92,19 @@ public void Convert_SingleLineSecret_PositionConverted()
 
```

Comment:
adding the new checks into the pre existing tests made most sense to me here, let me know if you think I should write a seperate test

Reply:
I would have added to the existing tests too.